app/src/main/res/values/strings.xml

```diff
@@ -74,64 +74,64 @@
     <string name="prefs_service">Service preferences</string>
     <string name="prefs_encryption">Encryption</string>
-    <string name="prefs_encryption_summary">Enable to encrypt data backups</string>
+    <string name="prefs_encryption_summary">Encrypts data backups</string>
     <string name="prefs_password">Encryption password</string>
-    <string name="prefs_password_summary">Set to empty to disable encryption</string>
-    <string name="prefs_passwordconfirmation">Encryption password confirmation</string>
+    <string name="prefs_password_summary">Leave empty to turn off encryption</string>
+    <string name="prefs_passwordconfirmation">Password encrypted</string>
     <string name="prefs_password_match_true">The passwords match</string>
-    <string name="prefs_password_match_false">The passwords doesn\'t match</string>
-    <string name="prefs_excludecache">Exclude app\'s cache</string>
-    <string name="prefs_excludecache_summary">Exclude application\'s cache from data backup</string>
-    <string name="prefs_externaldata">Backup external data</string>
-    <string name="prefs_externaldata_summary">Backs up and restores data that is placed on the external storage (e.g. /storage/emulated/0/Android/data/)</string>
-    <string name="prefs_deviceprotecteddata">Backup device protected data</string>
-    <string name="prefs_deviceprotecteddata_summary">Backs up and restores data that is placed in data/user_de</string>
+    <string name="prefs_password_match_false">The passwords don\'t match</string>
+    <string name="prefs_excludecache">Exclude the app\'s cache</string>
+    <string name="prefs_excludecache_summary">Exclude the app\'s cache from the data backup</string>
+    <string name="prefs_externaldata">Back up external data</string>
+    <string name="prefs_externaldata_summary">Backs up and restores data placed on the external storage (e.g. /storage/emulated/0/Android/data/)</string>
+    <string name="prefs_deviceprotecteddata">Back up device protected data</string>
+    <string name="prefs_deviceprotecteddata_summary">Backs up and restores data placed in data/user_de</string>
     <string name="prefs_copyselfapk">Copy own APK</string>
     <string name="prefs_copyselfapk_summary">Place a copy of OAndBackupX in the top level of the backup folder when backed up</string>
     <string name="prefs_acquirewakelock">Prevent sleep</string>
-    <string name="prefs_acquirewakelock_summary">Acquire a wakelock while batch operations are running to prevent sleep. will speed up backups and restores considerably but will consume more energy</string>
-    <string name="prefs_enableSpecialBackups">Enable special backups</string>
+    <string name="prefs_acquirewakelock_summary">Acquire a wakelock while batch operations are running to prevent sleep and speed up backups and restoring considerably by using more energy</string>
```

Comment:
"Keep the device on automatically for the duration of the operation to speed it up."